amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/model/hold/HoldBulkOperation.java

```diff
@@ -57,4 +57,27 @@ public enum HoldBulkOperationType
     private RestRequestQueryModel query;
     @JsonProperty(required = true)
     private HoldBulkOperationType op;
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o)
+        {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass())
+        {
+            return false;
+        }
+        HoldBulkOperation that = (HoldBulkOperation) o;
+        return Objects.equals(query.getQuery(), that.query.getQuery()) && Objects.equals(query.getUserQuery(),
```

Comment:
I suggest we just add an `equals` and `hashCode` to `RestRequestQueryModel` instead, and then use the Lombok annotation above.

Reply:
Done.